packages/hooks/src/useCFDAccounts.ts

```diff
@@ -0,0 +1,18 @@
+import { useStore } from '@deriv/stores';
+
+const useCFDAccounts = () => {
+    const { client } = useStore();
+    const { dxtrade_accounts_list, mt5_login_list } = client;
+
+    let cfd_accounts: typeof mt5_login_list = [];
+    if (Array.isArray(mt5_login_list)) {
+        cfd_accounts = [...cfd_accounts, ...mt5_login_list];
+    }
+    if (Array.isArray(dxtrade_accounts_list)) {
+        cfd_accounts = [...cfd_accounts, ...dxtrade_accounts_list];
+    }
```

Comment:
@shayan-deriv This part looks a bit weird, Maybe @thisyahlen-deriv can help you to understand the code and simplify it 🙏🏻

Reply:
sure man. will check with him